string connect_type
uint32 capture_count

